tests/job_tests/recognition/test_scoring.py

```diff
@@ -0,0 +1,109 @@
+import os
+import tempfile
+from typing import Optional
+
+from sisyphus import tk, setup_path
+
+from i6_core.tools.compile import MakeJob
+from i6_core.tools.git import CloneGitRepositoryJob
+from i6_core.recognition import ScliteJob
+
+Path = setup_path(__package__)
+
+
+def compile_sctk(

```

Comment:
This is taken from `i6_experiments/common/tools`, right? We might move it to i6_core, but then again, the effort is maybe unnecessary just for this test.

Reply:
Yes (though slightly modified for the test).
I think for now it should stay there.